src/main.js

```diff
@@ -1,6 +1,6 @@
 import { createApp } from 'vue';
 import { createPinia } from 'pinia';
-import rollbar from '@/plugins/rollbar';
+// import rollbar from '@/plugins/rollbar';
```

Comment:
To be fixed with #1129. I missed this stale import when pushing directly to main to temporarily fix that issue